### PR TITLE
New Deploy Command

### PR DESCRIPTION
**Pull request by kbartush**
Add a deploy command that allows you to do an initial config action, upload a contract and run initial transactions more easily.

### CODE REVIEW

File: src/mazzaroth.js [hunk 1]
```diff
@@ -431,4 +439,97 @@ subCmd.action(function (val, options) {
   process.stdin.on('data', process.exit.bind(process, 0))
 })
 
+const deployCmd = program.command('deploy <val>')
+const deployCmdDescription = `
+Helper for deploying a contract to a mazzaroth network. Takes a json config file,
+a sample config file can be found at
+https://github.com/kochavalabs/full-contract-example/blob/master/example_deploy.json
```
Comment (jeff1010322) [inline src/mazzaroth.js:446]: This link didn't work for me, looks like you renamed it to just deploy.json

File: src/mazzaroth.js [hunk 2]
```diff
@@ -431,4 +439,97 @@ subCmd.action(function (val, options) {
...
+
+Examples:
+  mazzaroth-cli deploy ./example_deploy.json
+`
+
+deployCmd.description(deployCmdDescription)
+deployCmd.action(async function (configPath) {
+  const config = JSON.parse(fs.readFileSync(configPath))
+  const channel = config['channel-id'] || defaultChannel
+  const host = config['node-addr'] || defaultAddr
```
Comment (jeff1010322) [inline src/mazzaroth.js:456]: Should host be in the config or included as an option on the command itself?  I would think the config should work without specifying the node address ahead of time and make it easier for deploying to different environments unless there is a reason I am not thinking of for it to need to be in the config.

File: src/mazzaroth.js (hunk 1 above)
Comment (kbartush) [reply src/mazzaroth.js]: Fixed

File: src/mazzaroth.js (hunk 2 above)
Comment (kbartush) [reply src/mazzaroth.js:456]: I prefer having it in the config. I think a similar argument can be made for pretty much everything in here and I like the consistency of having it one way or the other.

I don't feel to strongly about it though so I'll ad a CLI argument for host.

**Pull request merged**